src/lib/utils/pack.ts

```
@@ -48,7 +48,8 @@ export async function pack(srcDir: string, dstDir: string) {
         );
       });
 
-  const fileName = path.basename(stdout.trim());
+  const lastLine = stdout.trim().split("\n").pop() || "";
```

I think it would be better to throw a descriptive error if lastLine is undefined, don't you think? Maybe even add the stdout.trim() to the message, like `Failed to parse last line from stdout: ${stdout.trim()}`

If basename can't parse the empty string, or if fileName becomes an empty string, the resulting error might not be so obvious.

Also, I have a slight preference for `at(-1)` instead of `pop()`, but that's nitpicking :)

---

Agreed, updated now!